react-debugging.html

```diff
@@ -81,6 +83,7 @@ <h1>Draw a Card</h1>
               )
             })}
           </div>
+          </>
```

Comment:
Great effort! Take a look at  lines 71 and 74, what could be the cause of bugs and how can you resolve it?

Reply:
I am missing my function. The function will produce a random card every time the button is clicked. In the function, we'll create an array that includes all of the suits and their values. To ensure that  each click produces a random card, we'd use Math.floor & math.random. Then we could use string interpolation to inform the user of what card they've pulled.